kernel/fs/tmpfs.rs

```diff
@@ -170,6 +170,11 @@ impl Directory for Dir {
 
         Ok((inode as Arc<dyn Directory>).into())
     }
+
```

Comment:
ditto: Please revert changes in this file. Let's work on chmod implementation in tmpfs in a separate PR.